components/brave_wallet_ui/page/screens/send/components/account-list-item/account-list-item.tsx

```diff
@@ -18,20 +19,37 @@ interface Props {
   account: BraveWallet.AccountInfo
   onClick: (account: BraveWallet.AccountInfo) => void
   isSelected: boolean
+  selectedAccountKeyring: number | undefined
 }
 
 export const AccountListItem = (props: Props) => {
-  const { onClick, account, isSelected } = props
+  const { onClick, account, isSelected, selectedAccountKeyring } = props
 
   // hooks
   const orb = useAccountOrb(account)
 
+  const apiProxy = getAPIProxy()
+  const [fvmAddress, setFvmAddress] = React.useState<string>()
+  if (account.accountId.coin === BraveWallet.CoinType.ETH &&
+      (selectedAccountKeyring === BraveWallet.KeyringId.kFilecoin ||
+       selectedAccountKeyring === BraveWallet.KeyringId.kFilecoinTestnet)) {
+    apiProxy.braveWalletService.convertFEVMToFVMAddress(
+      account.accountId.keyringId === BraveWallet.KeyringId.kFilecoin, account.address).then(v => {
+        if (v.result) {
+          setFvmAddress(v.result)
+        }
+      })
+  }
+
```

Comment:
We need to implement a "slice" for `convertFEVMToFVMAddress` in `api.slice.ts` and access it here using the hooks generated by Redux Toolkit. Let me know if you need help with this, and I can provide some pointers.